### PR TITLE
Fixes #2: Create CONTRIBUTING.md

### LINKED ISSUE #2: Translation via Transifex
I have setup a transifex project to continue translating.

Repo: https://www.transifex.com/alir3z4/flarum-extension-persian

It will be automatically updated whenever the English locales get updated.
I have sent an invite to your email to be a maintainer on the translation project.

### PR DESCRIPTION
**Pull request by Alir3z4**
Fixes #2

### CODE REVIEW

**Comment by Alir3z4** [comment]
writing Persian in the online editor is hard, I had to make the commits dirty.
Please squash to not mess with the commit history.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div dir="rtl"># مشارکت</div>
```
Comment (sijad) [inline CONTRIBUTING.md]: I think this should be `<h1 dir="rtl">مشارکت</h1>` instead.

Comment (Alir3z4) [reply CONTRIBUTING.md]: let me try that and see how it goes

Comment (Alir3z4) [reply CONTRIBUTING.md]: You're right, it worked.
Have a look again please ;)

**Comment by sijad** [comment]
Thank you!

**Comment by Alir3z4** [comment]
No problem, glad it has been merged

On Apr 14, 2018 19:38, "Sajjad Hashemian" <notifications@github.com> wrote:

Thank you!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/sijad/flarum-ext-persian/pull/4#issuecomment-381342062>,
or mute the thread

**Pull request merged**